tests/js/server/recovery/view-arangosearch-link-populate-persistentids-mmfiles.js

```diff
@@ -74,24 +72,17 @@ function recoverySuite () {
 
       var meta = { links: { 'UnitTestsRecoveryDummy': { includeAllFields: true } } };
 
-      let expectFailure = internal.debugCanUseFailAt();
       try {
         view.properties(meta);
-        assertFalse(expectFailure);
+        fail();
       } catch (e) {
-        internal.print(JSON.stringify(e))
-        if (e.errorNum !== arangodb.ERROR_ARANGO_INDEX_CREATION_FAILED){
-          //throw e;
+        if (e.errorNum !== errors.ERROR_ARANGO_INDEX_CREATION_FAILED.code) {
+          throw e;
         }
       }
 
       var links = view.properties().links;
-      if (expectFailure) {
-        assertEqual(links['UnitTestsRecoveryDummy'], undefined);
-      } else {
-        assertNotEqual(links['UnitTestsRecoveryDummy'], undefined);
-        assertTrue(links['UnitTestsRecoveryDummy'].includeAllFields);
-      }
+      assertEqual(links['UnitTestsRecoveryDummy'], undefined);
```

Comment:
instead of using `assertEqual(..., undefined)`, the more intuitive variant would be `assertUndefined(...)`.